spark/src/main/scala/geotrellis/spark/tiling/ZoomedLayoutScheme.scala

```diff
@@ -33,42 +33,52 @@ class ZoomedLayoutScheme(tileSize: Int) extends LayoutScheme {
   /** TODO: Improve this algorithm. One improvement is to follow the algorithm
     * described in  "Tile-Based Geospatial Information Systems Principles and Practices"
     * by John T. Sample & Elias Ioup, section 3.1.2 */
-  def levelFor(worldExtent: Extent, cellSize: CellSize): LayoutLevel = {
+  def levelFor(projectedExtent: ProjectedExtent, cellSize: CellSize) = {
```

Comment:
Do you mean that the extent is unused here? Basically this allows the "fork" in behavior between FloatingLayout and Zoomed layout:
https://github.com/geotrellis/geotrellis/pull/1161/files#diff-13f7f1feac59e88edb81e3998f049605R15

Zoomed layout uses the CRS (to get world extent), FloatingLayout uses the extent.

This seems like the cleanest way to unify the interface.